src/System.Net.Http/tests/FunctionalTests/HttpProtocolTests.cs

```diff
@@ -15,6 +15,8 @@ public class HttpProtocolTests : HttpClientTest
         protected virtual Stream GetStream(Stream s) => s;
 
         [Theory]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Requires fix shipping in .NET 4.7.3")]
+        [InlineData("HTTP/1.1 200", 200, "")] // This is the test data requires the fix in .NET Framework 4.7.3
```

Comment:
I think the attribute will apply to all test cases. Can you please split it into 2 parts - one with this one and one with the rest? Or just merge it with your GetAsync_StatusLineNotFollowRFC_SuccessOnCore for now + add comment on this particular test case it will be fixed in 4.7.3.

Reply:
Will fix.